### PR TITLE
Add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by kenjis**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
+about: How to submit a feature request
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please submit feature requests to [Discussions](https://github.com/codeigniter4/shield/discussions/categories/ideas).
```
Comment (MGatner) [inline .github/ISSUE_TEMPLATE/feature_request.md]: This should not be Shield-specific.

**Review by MGatner — CHANGES_REQUESTED** [review]
This is the first change that really seems to make the decision that this repo is a template for **the CodeIgniter Organization**, not for any devs working with CodeIgniter in their own spaces. I'm not opposed but want to be explicit about that choice.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (kenjis) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Thanks! Fixed.

**Comment by MGatner** [comment]
@kenjis do you see this repo as an internal tool for the CodeIgniter organization?

**Comment by kenjis** [comment]
Yes! This repo is for the CodeIgniter organization.
Not for everyone who uses CI4.

**Comment by kenjis** [comment]
@MGatner I sent #13

**Review by MGatner — APPROVED** [review]

**Pull request merged**